collection.js

```diff
@@ -0,0 +1,41 @@
+const LOCAL_STORAGE_KEY = 'books';
+const list = document.getElementById('books');
+
+const getFromLocalStorage = () => {
+  const cachedBooks = window.localStorage.getItem(LOCAL_STORAGE_KEY);
+  return cachedBooks ? JSON.parse(cachedBooks) : [];
+};
+
+let booksArr = getFromLocalStorage();
+
+const saveToLocalStorage = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(booksArr));
+};
+
+function render() {
+  list.innerHTML = booksArr.map((book) => (`<li>
+            <article>
+                <p id="title">${book.title}</p>
+                <p id="author">${book.author}</p>
+                <button class="remove">Remove</button>
+            </article>`)).join('');
+
+  const remove = Array.from(document.getElementsByClassName('remove'));
+  remove.forEach((btn, i) => btn.addEventListener('click', () => {
+    booksArr = booksArr.filter((book, ind) => i !== ind);
```

Comment:
- [ ] Same here, as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).), Please create a separate function for removing a book from a list of books and you can call it where ever you need.